src/lib.rs

```diff
@@ -130,6 +130,8 @@ pub use self::filters::{
     query,
     // query() function
     query::query,
+    // raw_query() function
+    query::raw_query,
```

Comment:
This is just a thought, but what do you think if the function were just `raw`, and not re-exported, just left in the `query` module? That'd mean typical usage would then be `warp::query::raw()`...

Reply:
Makes sense to me, this way it won't seem like a top-level filter. I'll make the change.